src/index.js

```diff
@@ -470,6 +470,9 @@ function createConfig(options, entry, format, writeMeta) {
 									preset: 'default',
 								}),
 						].filter(Boolean),
+						modules:
```

Comment:
We should also document that `microbundle` already supports `foo.module.css` for enabling CSS modules per-file, since [rollup-plugin-postcss' autoModules option](https://github.com/egoist/rollup-plugin-postcss#automodules) defaults to `true`.

In that case we could potentially turn `--css-modules` into a tristate flag (where the default value is `null`):

| Command | What happens when it is run |
|-|-|
| `microbundle` | Allow CSS modules via `.module.css`
| `microbundle --css-modules` | Turn on CSS modules for all `.css` files
| `microbundle --css-modules false` | Disable CSS modules, even for `.module.css` files
| `microbundle --css-modules "_[hash]"` | Turn on CSS modules for all `.css` files<br>**and** provide custom name generation